development/openstack-base-focal-ussuri-ovn/README.md

```diff
@@ -12,7 +12,7 @@ For example, a section similar to this exists in the bundle.yaml file.  The thir
 
 ```
 variables:
-  openstack-origin:    &openstack-origin     cloud:eoan-train
+  openstack-origin:    &openstack-origin     distro
```

Comment:
PR #170 uses distro-proposed per our release notes: https://docs.openstack.org/charm-guide/latest/2005.html#openstack-ussuri-packages-for-ubuntu-20-04-lts

Should we do the same here?

Reply:
Yes, good catch. Updated.